### PR TITLE
Fill the translations for locale ja.

### PR DESCRIPTION
**Pull request by taizan-hokuto**
UIの日本語localeファイルを入力しました。

### CODE REVIEW

File: src/popup/i18n/locales/ja.json5 [hunk 1]
```diff
@@ -30,7 +30,7 @@
         },
       },
       // Display the current version
-      version: "Version: v{version}",
+      version: "バージョン: v{version}",
       // Describe the backend engine of this project
       engine: {
         // Template used to construct the description for the engine
```
Comment (YunzheZJU) [inline src/popup/i18n/locales/ja.json5:36]: Is the next line keep untranslated intentionally?
```
label: "Powered by {link}",
```

Comment (taizan-hokuto) [reply src/popup/i18n/locales/ja.json5:36]: Yes. I couldn't think of any Japanese words that fit. The phrase "Powered by" is natural and cooler to keep in English.

Comment (YunzheZJU) [reply src/popup/i18n/locales/ja.json5:36]: > Yes. I couldn't think of any Japanese words that fit. The phrase "Powered by" is natural and cooler to keep in English. 

Cool! That's great!

File: src/background/i18n/locales/ja.json5 [hunk 2]
```diff
@@ -2,8 +2,8 @@
   // Desktop notification
   notification: {
     // Inform users that a guerrilla live has started
-    guerrilla: 'A guerrilla live has started! {name} is waiting for your watching!',
+    ゲリラライブ: '{name} がゲリラライブを開始しました！',
     // Inform users that a scheduled live has started
-    reminder: 'A scheduled live has started! {name} is waiting for your watching!',
+    リマインダ: '{name} が配信を開始しました！',
```
[the remark this answers is not in the document]
Comment (taizan-hokuto) [reply src/background/i18n/locales/ja.json5]: Fixed them.

**Comment by YunzheZJU** [comment]
本当にありがとうございます！

**Pull request merged**